pkg/admin/graphql/resource.go

```diff
@@ -36,6 +36,21 @@ var nodeResource = node(
 				Type:        graphql.String,
 				Description: "The optional name of the resource.",
 			},
+			"clientIDs": &graphql.Field{
+				Type:        graphql.NewNonNull(graphql.NewList(graphql.NewNonNull(graphql.String))),
+				Description: "The list of client IDs associated with this Resource.",
+				Resolve: func(p graphql.ResolveParams) (interface{}, error) {
+					resource := p.Source.(*model.Resource)
+					ctx := p.Context
+					gqlCtx := GQLContext(ctx)
+
+					clientIDs, err := gqlCtx.ResourceScopeFacade.ListClientIDsByResourceID(ctx, resource.ID)
```

Comment:
When the query fetches a list of resources, and the query fetches the field `clientIDs`, then it is N+1 query pattern.

I notice we also have this pattern in User.identities, user.authenticators, etc.

For this particular instance of this pattern, maybe we introduce a loader to load the client IDs of resource ID.